third_party/transducer/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.5)
+
+PROJECT(rnnt_release)
+
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2")
+
+IF(APPLE)
+    ADD_DEFINITIONS(-DAPPLE)
+ENDIF()
+
+INCLUDE_DIRECTORIES(submodule/include)
+
+SET(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+ADD_DEFINITIONS(-DRNNT_DISABLE_OMP)
+
+IF(APPLE)
+    EXEC_PROGRAM(uname ARGS -v  OUTPUT_VARIABLE DARWIN_VERSION)
+    STRING(REGEX MATCH "[0-9]+" DARWIN_VERSION ${DARWIN_VERSION})
+    MESSAGE(STATUS "DARWIN_VERSION=${DARWIN_VERSION}")
+
+    # for el capitain have to use rpath
+    IF(DARWIN_VERSION LESS 15)
+        SET(CMAKE_SKIP_RPATH TRUE)
+    ENDIF()
+
+ELSE()
+    # always skip for linux
+    SET(CMAKE_SKIP_RPATH TRUE)
+ENDIF()
+
+ADD_LIBRARY(warprnnt STATIC submodule/src/rnnt_entrypoint.cpp)
+
+INSTALL(TARGETS warprnnt
+        LIBRARY DESTINATION "lib"
+        ARCHIVE DESTINATION "archives")
+
+INSTALL(FILES submodule/include/rnnt.h DESTINATION "submodule/include")
```

Comment:
"archives" directory looks uncommon. Typically, you can use `"lib"`. however, as I am suggesting in the previous comments, installing the resulting library and header in `third_party/install/[lib|include]` will make the dependency management simpler.

For that you can remove `PROJECT` command from this file, and set `CMAKE_INSTALL_PREFIX` to `${CMAKE_CURRENT_SOURCE_DIR}/install`, then just `install(TARGETS warprnnt)` as the default subdirectory for static libraries (`archive`) is `"lib"`.